geometry/eolearn/geometry/transformations.py

```diff
@@ -200,19 +200,22 @@ def execute(self, eopatch):
         if eopatch.bbox is None:
             raise ValueError('EOPatch has to have a bounding box')
 
+        feature_type, feature_name = self.raster_feature
```

Comment:
In the new version of eo-learn you don't have to do this anymore. You can simply use `eopatch[self.raster_feature]` instead of `eopatch[feature_type][feature_name]`